src/app/content/components/Course.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Heading, Text } from '@chakra-ui/layout';
+import { BackgroundProps, Divider, Flex } from '@chakra-ui/react';
+import React, { PropsWithChildren } from 'react';
+
+export interface CourseShieldProps {
+  title: string;
+  bg?: BackgroundProps['bg'];
+}
+
+export function CourseShield({ title, children, bg }: PropsWithChildren<CourseShieldProps>): JSX.Element {
+  return (
+    <Flex m="1">
+      <Box bg="gray.100" p="1" borderTopLeftRadius="md" borderBottomLeftRadius="md">
+        <Heading size="md" color="gray.600" px="3">
+          {title}
+        </Heading>
+      </Box>
+      <Flex bg={bg} p="1" px="3" borderBottomRightRadius="md" borderTopRightRadius="md">
+        {children}
+      </Flex>
+    </Flex>
+  );
+}
+
+export interface CourseInfoProps {
+  subject: string;
+  code: string;
+  title: string;
+  description: string;
+  hours?: { lecture: string; tutorial: string; lab: string };
+  credits?: string;
+  addtionalNotes?: string;
+  units?: string;
+}
+
+export function CourseInfo({
```

Comment:
Not sure if we are going to have a footer or not.. but if we don't we need to add some padding/margin to the bottom. 

![Screen Shot 2021-03-07 at 8 59 56 PM](https://user-images.githubusercontent.com/31324580/110276744-affff880-7f88-11eb-91b1-c4b96ef0a97d.png)